### DISCUSSION TITLE
[Bug] - Month issue

### DISCUSSION THREAD

**Discussion by ERRon2** [Monika-After-Story/MonikaModDev#9248] [QA - tech support + general] [answered]
So, my Monika thinks it's winter right now, but (for me anyway) it's summer. I believe this is a small bug, but I don't know. I don't even know if I'm putting this bug into the right area, I can hope I am though. But the bug is that Monika thinks it's winter when It's summer, My Email is ... If you cant use that Email then MY Email is ... But I don't use this one really hah...
If you need a picture of the bug, tell me and I can send it to you.

**Comment by Booplicate** [comment, marked as answer]
You probably told her you live in the south hemisphere. Talk about hemispheres again and select north.

**Comment by ERRon2** [comment]
Ohhhhhh
Okay thanks

**Comment by ERRon2** [comment]
sorry again...
